src/core/webrtc/simple-room.ts

```diff
@@ -130,6 +130,20 @@ export const getRoom = (id: string) => {
     RoomEvent.TrackStreamStateChanged,
   ]
 
+  /* Subscribing to the RoomEvent.DataReceived event.
+   * Update the room participant info when Participant metadata gets updated
+   */
+  room.subscribeToRoomEvent(
+    RoomEvent.DataReceived,
+    (payload, participant: Participant, kind: DataPacket_Kind) => {
+      const strData = decoder.decode(payload)
+      const data = JSON.parse(strData)
+      if (data.type === 'ParticipantMetadataUpdate') {
```

Comment:
Probably change the reference from the magic string to `DataType.ParticipantMetadataUpdate`